db/packet_db.txt

```diff
@@ -2498,6 +2498,7 @@ packet_keys: 0x62C86D09,0x75944F17,0x112C133D // [YomRawr]
 0x0A32,2		// ZC_OPEN_RODEX_THROUGH_NPC_ONLY
 0x0A13,26,dull,0	// CZ_CHECK_RECEIVE_CHARACTER_NAME
 0x0A14,10		// ZC_CHECK_RECEIVE_CHARACTER_NAME
+0x0A3B,-1,ZC_HAT_EFFECT,2:4:8:9
```

Comment:
Hmm.
Afaik there was issue when packet not recognized for recent packet after older one.
Can't remember the cause, because of multi-client, or because of using `packet_len(x)`.
But I'll try to remove it, I didn't test for recent version except for 2015-09-16 yet.